src/pages/Movies.js

```diff
@@ -9,12 +9,13 @@ import { movieActions } from "../store/data-slice";
 
 const Movies = () => {
   const data = useSelector((state) => state.movies.data);
+  const isLoading=useSelector((state)=>state.movies.isLoading)
   const card = (item) => {
     return <MovieCard movie={item} key={item.id} />;
```

Comment:
same for `key={item.id`

Reply:
I don't quite understand what you mean. I can't find any suitable position fir key in ShowList component.